docs/Installing_IML_on_HPC_Storage_Sandbox.md

```diff
@@ -0,0 +1,143 @@
+
+## Prerequisites:
+Please refer to https://github.com/intel-hpdd/vagrantfiles on how to create a virtual HPC storage cluster with vagrant before attempting to install IML.
+
+## Notes:
+- You will be logging into your vagrant box as the vagrant user, which has the ability to run with root privilege. To elevate privileges to the root account, use the sudo command. The vagrant user does not require a password to run sudo. Should there ever be a need to login as root directly, the root password is also "vagrant". 
+
+    To acquire a root shell, use sudo in either one of the following invocations:
+    ```
+    sudo -s
+    ```
+    or
+    ```
+    sudo su -
+    ```
+
+## Installing IML:
+1. Verify the following vagrant plugins are installed:
+    ```
+    vagrant plugin install vagrant-shell-commander
+    ```
+2. Obtain the [preview 3 build](https://github.com/intel-hpdd/intel-manager-for-lustre/releases/tag/v4.0.0.0P3).
+3. Exit from the vagrant box and scp the build to the /tmp directory in your admin node. For example, if your admin node is running on port 2200 (you can verify this with `vagrant ssh-config`) and the build is in your Downloads folder:
+    ```
+    scp -P 2200 ~/Downloads/iml-4.0.0.0.tar.gz vagrant@127.0.0.1:/tmp/.
+    # password is "vagrant"
+    ```
+    Alternatively, use the output of `vagrant ssh-config` to create a temporary SSH configuration for the `scp` command. This is most useful when there will be a lot of interaction with a node from outside the VM:
+    ```
+    [user@mini iml]$ vagrant ssh-config | tee sshcfg| grep ^Host
+    Host adm
+    Host mds1
+    ...
+    [user@mini iml]$ scp -F sshcfg ~/Media/iml-4.0.0.0.tar.gz adm:/tmp/.
+    iml-4.0.0.0.tar.gz                                                 100%  130MB  69.5MB/s   00:01    
+    ```
+    This allows you to use the Host name referenced in the ssh config to identify individual nodes in the Vagrant environment, and means you do not have to track the port numbers or use a password. In the above example, the file is copied on to the `adm` VM.
+
+4. ssh into the admin box and install the build:
+    ```
+    vagrant ssh
+    [vagrant@ct7-adm ~]$ sudo su - # (or "sudo -s")
+    [vagrant@ct7-adm ~]# cd /tmp
+    [vagrant@ct7-adm ~]# tar xvf <buildname>.tar.gz
+    [vagrant@ct7-adm ~]# cd <build folder>
+    [vagrant@ct7-adm ~]# ./install --no-dbspace-check
+    ```
+5. Update the /etc/hosts file on your computer to include the following line:
+    ```
+    127.0.0.1 ct7-adm.lfs.local
+    ```
+6. Finally, test that a connection can be made to IML by going to the following link in your browser:
+https://ct7-adm.lfs.local:8443
+
+## Adding Servers
+You should now be able to see IML when navigating to https://ct7-adm.lfs.local:8443. Click on the login link at the top right and log in as the admin. Next, go to the server configuration page and add the following servers:
+```
+ct7-mds[1,2].lfs.local,ct7-oss[1,2].lfs.local
```

Comment:
ct7 prefixes have been removed. This is now:

`mds[1,2].lfs.local,oss[1,2].lfs.local`

Reply:
done